lib/webpacker/railtie.rb

```diff
@@ -1,26 +1,24 @@
-require 'rails/railtie'
+require "rails/railtie"
 
-require 'webpacker/helper'
-require 'webpacker/digests'
+require "webpacker/helper"
 
 class Webpacker::Engine < ::Rails::Engine
   initializer :webpacker do |app|
     ActiveSupport.on_load :action_controller do
       ActionController::Base.helper Webpacker::Helper
     end
 
-    app.config.x.webpacker[:packs_dist_dir] ||= 'packs'
-    app.config.x.webpacker[:packs_dist_path] ||= \
-      "/#{app.config.x.webpacker[:packs_dist_dir]}"
+    # Load config from package.json or initialise with defaults
+    # when running rails webpacker:install
+    Webpacker::Configuration.load
+    webpacker_config = Webpacker::Configuration.webpacker
 
-    if app.config.x.webpacker[:digesting]
-      app.config.x.webpacker[:digests_path] ||= \
-        Rails.root.join('public',
-                        app.config.x.webpacker[:packs_dist_dir],
-                        'digests.json')
-
-      Webpacker::Digests.load \
-        app.config.x.webpacker[:digests_path]
+    if !(webpacker_config && webpacker_config[:distPath])
+      webpacker_config = { distPath: "public/packs", manifestFileName: "manifest.json" }
     end
+
+    Webpacker::Manifest.load(
+      Rails.root.join(webpacker_config[:distPath], webpacker_config[:manifestFileName])
+    )
```

Comment:
Manifest should be able to load from config alone. Odd to do this level of work here. So should be something like: `Webpacker::Manifest.load(Webpacker::Configuration.manifest_path)`.